### DISCUSSION TITLE
Disabled buildings to count as Mothballed

### DISCUSSION THREAD

**Discussion by catalin-duta** [Keriew/augustus#517] [Ideas]
For the construction of a Grand Temple, I had to build more Marble Quarries than generally used for export or large temples.
Once the marble was fully supplied, I disabled the extra quarries, keeping active just the 6 I had before.
At the same time, due to previous overproduction, I went to the Trade Advisor to put a temporary stop while traders buy the extra stock.
The advisor only lists the 6 non-disabled industries.
I'm thinking in a big city it could be easy to forget there even are disabled industries somewhere, if the advisor doesn't know about them at all.

**Comment by Ouaz** [comment]
> I'm thinking in a big city it could be easy to forget there even are disabled industries somewhere, if the advisor doesn't know about them at all.

The Trade Advisor only counts the number of working industries or the total number of mothballed industries when it is ordered by him.
I don't know how much work it would require to implement the number of individual mothballed industries on the Trade Advisor screen, either by adding a new line (_3 individual mothballed industries in the city_) or by adding the total number of industries (_6 / 9 working industries in the city_), but it may be redundant / confusing with idle industries in the city. Seems tricky.

However, you can keep track of individual mothballed industries by using the Mothballed overlay (in Trade overlays).

**Reply by catalin-duta** [reply]
I understand now. I thought there was only 1 state for mothballed, not 2 (via advisor and manually). I also didn't know about the overlay.

Still, maybe it would be useful in the future to also list the disabled industries in the Trade tab and be able to restart all of the industry from there.

**Reply by crudelios** [reply]
This is a good suggestion.
